package.json

```diff
@@ -101,7 +101,11 @@
     "ejs": "^3.1.7",
     "cross-fetch": "^3.1.5",
     "eventsource": "^2.0.2",
-    "protobufjs": "^6.11.3"
+    "protobufjs": "^6.11.3",
+    "shell-quote": "^1.7.3",
```

Comment:
Where were these 3 dependencies added

Reply:
I didn't see any new import statements with these why do we need it

Reply:
This is to fixing errors and warnings when running `yarn audit`